### DISCUSSION TITLE
Send chat as multiple messages

### DISCUSSION THREAD

**Discussion by vatsake** [runelite/runelite#18035] [Ideas]
When the limit is reached, instead of restricting typing, maybe send multiple messages instead?

**Comment by Felanbird** [comment]
I think this would very likely be against the rules as it would be `1 enter press = multiple actions sent to the server`, so I don't believe this would be possible even though your explicit usage is just as minimal as sending a chat message

**Reply by vatsake** [reply]
Maybe they will overlook it? 😉
